realm/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -39,7 +38,7 @@
     private static final String JAVA_LIBRARY_PATH = "java.library.path";
 //*/
 
-    private static AtomicBoolean libraryIsLoaded = new AtomicBoolean(false);
+    private static boolean libraryIsLoaded = false;
```

Comment:
Tracked down the source in AOSP: http://osxr.org/android/source/dalvik/vm/Native.cpp#0318
Boolean is not required :)

If that is removed. It looks fine to me :+1: